e2e-tests/ukef/cypress/support/portal/logIn.js

```diff
@@ -15,5 +16,9 @@ module.exports = (opts) => {
     signInLink.visit({ token: signInToken, userId: _id });
   });
 
-  cy.url().should('eq', relative('/dashboard/deals/0'));
+  if (roles?.length === 1 && roles?.at(0) === ROLES.PAYMENT_REPORT_OFFICER) {
```

Comment:
Is `roles` an array of strings? If so I think this would be better, just incase the ordering changes:

```js
if (roles.includes(ROLES.PAYMENT_REPORT_OFFICER)) { ... }
```

What do you think?

Reply:
FWIW I think the issue here is we want to test the redirect when your role is ONLY payment_report_officer, not when it's one of your multiple roles and one of your roles in `payment_report_officer` which means we need the extra checks

Reply:
Can we add a comment for this please, it's not obvious otherwise

My concern was that if the `roles` param changes in some relevant test, to have a different first item or multiple roles, this'd fail

Reply:
Added a comment 👍